chcemvediet/static/main/css/custom.sass

```diff
@@ -80,6 +80,50 @@ a:hover, a:focus
       padding-top: chv_baseline_fix($chv-fontsize-base, 1)
       overflow: hidden
 
+//
+// Casestudy link component
+//
+
+.chv-casestudy.chv-casestudy
+  display: block
+  padding: ($chv-baseline - 1px)
+  margin-bottom: ($chv-baseline * 1.2)
```

Comment:
Aby nenastalo zle zalomenie ako na obrazku [sem](https://github.com/martinmacko47/chcemvediet/pull/370/files#r616077278), je potrebne aby bol stpec "Vybrane pripady" vyssi (rovnako velky) ako stlpec "O com to je". Fungovat to bude za predpokladu ze sa velkost uz nebude mennit, teda zmensovat.

Reply:
Zalomenie `<div class="col-*">` boxov v gride by som neriesil cez presne vysky stlpcov, to je dost nachylne na zmeny/chyby.

Ak niekde treba vynutit zalomenie `<div class="col-*">`, tak su v principe dva riesenia:
a) Niektore `<div class="col-*">` zabalit do najprv do jedneho sirsieho stlpca a potom vo vnutri toho stplca pouzit `<div class="row">` s tymi stlpcami. Tym sa da vynutit, ze sa tieto stlpce v ramci jedneho `<div class="row">` vzdy zalomia spolu.
b) Medzi dva `<div class="col-*">` ktore chceme zalomit vlozit prazdny `<div class="col-*-12">`. Kedze `col-*-12` je na celu sirku, to vzdy vynuti zalomenie. Pomocou spravneho rezu maniesto `*`, pripadne pomocou podmieneneho (ne)zobrazenia toho prazdneho boxu, vieme nastavit, pri ktorych rezoch sa to zalomenie vynuti a pri ktorych nie.

Reply:
:ok: pouzil som b)